FrEee.Tests/post-build.bat

```diff
@@ -4,9 +4,9 @@ for /f "tokens=1,* delims= " %%a in ("%*") do set ALL_BUT_FIRST=%%b
 
 echo Copying assets to %ALL_BUT_FIRST%\bin\%1\net472
 
-robocopy "%ALL_BUT_FIRST%\..\FrEee\Mods" "%ALL_BUT_FIRST%\bin\%1\Mods" /e
-robocopy "%ALL_BUT_FIRST%\..\FrEee\Data" "%ALL_BUT_FIRST%\bin\%1\Data" /e
-robocopy "%ALL_BUT_FIRST%\Mods" "%ALL_BUT_FIRST%\bin\%1\Mods" /e
-robocopy "%ALL_BUT_FIRST%\Savegame" "%ALL_BUT_FIRST%\bin\%1\Savegame" /e
+robocopy "%ALL_BUT_FIRST%\..\FrEee\Mods" "%ALL_BUT_FIRST%\%1\Mods" /e
+robocopy "%ALL_BUT_FIRST%\..\FrEee\Data" "%ALL_BUT_FIRST%\%1\Data" /e
+robocopy "%ALL_BUT_FIRST%\Mods" "%ALL_BUT_FIRST%\%1\Mods" /e
+robocopy "%ALL_BUT_FIRST%\Savegame" "%ALL_BUT_FIRST%\%1\Savegame" /e

```

Comment:
Why is there no longer a bin folder?

Reply:
I'm not sure what was passed as the %1 parameter, but now it's $(OutDir) macro in post-built event, which includes 'bin' part of the path already, so I've removed it from the path to avoid duplication.